modules/packages/Curl.chpl

```diff
@@ -36,711 +38,1040 @@ The `curl homepage <https://curl.haxx.se/libcurl/>`_ describes libcurl thus::
 Dependencies
 ------------
 
-The Curl module in Chapel is dependent on libcurl. For more information
- on how to setup libcurl, see the :ref:`readme-auxIO` page.
+The Curl module in Chapel is dependent on libcurl being installed and available
+to Chapel pograms.
 
 Using Curl Support in Chapel
```

Comment:
Seems odd that the only Curl examples shown are with the `URL` module. Would it be useful to show a standalone `Curl` example or no?

Reply:
It could be but I don't really know how to write a useful one & my focus has been on the I/O integration. I don't know what a useful use case for it would be.